modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v4/form_data.rb

```diff
@@ -47,6 +46,13 @@ class FormData
             'OTHER'
           ].freeze
 
+          MST_EVENT_DISCLOSURE = {
```

Comment:
Not blocking: this object might benefit from being sorted by order of the integer value, similar to `BENEFIT_TYPE_CODES` et al.

Reply:
Ok, so, there is SOME order to what i did here, the ordering I used was the order on the PDF form itself, so, yea, the integer in the hash is the index into the actual groupbox "control" on the form.  IDK why these indexes don't match the order that they appear on the form....i've heard that ~pdfs don't have a lot of ordering logic, it's mostly just position on page...and it's just trial and error to figure out the index orders, i can't really see the order by inspecting the PDFs properties.
<img width="1039" alt="image" src="https://github.com/user-attachments/assets/88a72d66-983e-4a02-8f04-8453d9e8728e" />